core/kazoo_data/src/kzs_attachments.erl

```diff
@@ -198,16 +198,30 @@ handle_put_attachment(#{att_post_handler := 'external'}=Map, Att, DbName, DocId,
 external_attachment(Map, DbName, JObj, Att, Props) ->
     Atts = kz_json:merge_jobjs(Att, kz_json:get_value(?KEY_STUB_ATTACHMENTS, JObj, kz_json:new())),
     case kzs_doc:save_doc(Map, DbName, kz_json:set_values([{?KEY_STUB_ATTACHMENTS, Atts}], JObj), []) of
-        {'ok', Doc} -> {'ok', Doc, Props};
+        {'ok', Doc} ->
+            kzs_publish:maybe_publish_doc(DbName, JObj, Doc),
```

Comment:
@jamesaimonetti isn't this creating a `double` firing issue ?

Reply:
@lazedo yeah looks like it would. I'll PR a fix in a few mins

Reply:
@lazedo https://github.com/2600hz/kazoo/pull/5898